test/mobile/features/fixtures/Assets/Scripts/BugsnagNative.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.InteropServices; 
+using UnityEngine;
+
+public class BugsnagNative {
```

Comment:
Can the tabs in this file be replaced with spaces?

Reply:
Fixed here and in a couple of other files under `test/mobile`.